explorer/README.md

```diff
@@ -0,0 +1,82 @@
+# ETCExplorer
```

Comment:
This readme is kind of out of date in our version. Could you add a paragraph about running with docker-compose?

Reply:
Could you also add here some instructions on how to use a different config file? Presumably it's just a mounted volume in the `docker-compose.yml` , but it's currently unclear.

Reply:
yeah, not sure how we should mount that file.

The config file is on the original readme